docs/src/index.md

```diff
@@ -0,0 +1,64 @@
+# Games.jl
+
+*Games.jl* is a [`Julia`](http://www.julialang.org) package for computation of Game Theory.
+
+## Installation
+
+*Games.jl* is an unregistered package that currently under development.
+
+To install the package, open a `Julia` session and type
+
+```julia
+Pkg.clone("https://github.com/QuantEcon/Games.jl")
+```
+
+## Usage
+
+Once installed, the `Games` package can be used by typing
+
+```@example 1
+using Games
+```
+
+The Base type `Player` can be created by passing a payoff matrix.
+
+```@example 1
+player1 = Player([3 1; 0 2])
+```
+
+A 2-player `NormalFormGame` can be created either by passing `Player` instances.
```

Comment:
A 2-player `NormalFormGame` can be created either by passing `Player` instances**,**